packages/@expo/cli/src/export/__tests__/createKeepFile.test.ts

```diff
@@ -0,0 +1,44 @@
+import { vol } from 'memfs';
+import type { AssetData } from 'metro';
+
+import { createKeepFile } from '../persistMetroAssets';
+
+jest.mock('fs');
+
+describe(createKeepFile, () => {
+  const assets: AssetData[] = [
+    {
+      __packager_asset: true,
+      fileSystemLocation: 'input',
+      httpServerLocation: '/assets/input',
+      width: 100,
+      height: 100,
+      scales: [1, 2, 3],
+      hash: 'hash',
+      name: 'image',
+      type: 'png',
+      files: ['/input/image.png'],
+    },
+    {
+      __packager_asset: true,
+      fileSystemLocation: 'input',
+      httpServerLocation: '/assets/input',
+      width: undefined,
+      height: undefined,
+      scales: [1],
+      hash: 'hash',
+      name: 'testFont',
+      type: 'ttf',
+      files: ['/input/someFont.ttf'],
+    },
+  ];
+
+  it('creates a keep.xml file', async () => {
+    await createKeepFile(assets, '/output');
+
+    expect(vol.readdirSync('/output/raw')).toEqual(['keep.xml']);
```

Comment:
```suggestion
    expect(vol.existsSync('/output/raw/keep.xml')).toBe(true);
```

existsSync here is performant than readdir